OMEdit closes when simulating simple models of the MSL.

### Description
OMEdit closes when simulating simple models of the MSL in MacOS with ARM. These models can be successfully be built only, but not simulate. 

### Steps to reproduce
I have installed OM in MacOS with ARM from the latest source as in e8d0d2d92344b976a324c811671968516b39e83d and following https://github.com/OpenModelica/OpenModelica/discussions/9129. Summarizing, the steps were:
```ssh
brew install gettext boost autconf ccache OpenSceneGraph Qt5
export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk8u333.jdk
cd OpenModelica
mkdir build_cmake && cd build_cmake
cmake .. -Wno-dev -DOM_OMC_ENABLE_FORTRAN=OFF -DOM_OMC_ENABLE_IPOPT=OFF -DOM_OMC_ENABLE_CPP_RUNTIME=OFF -DCMAKE_PREFIX_PATH=/opt/local -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang |  tee cmake_config.txt
make -j8
make install
```
Then I can simulate some simple examples of the MSL like `Modelica.Blocks.Examples.Filter`. However, there are others like `Modelica.Blocks.Examples.PID_Controller` that I can build but not simulate. `OMEdit` would crash right before simulating. 

The following `reproduceBug.mos` is my minimal working example:
```modelica
getVersion(); getErrorString();
loadModel(Modelica,{"4.0"}); getErrorString();
simulate(Modelica.Blocks.Examples.Filter); getErrorString();
simulate(Modelica.Blocks.Examples.PID_Controller); getErrorString();
```
of which the output is:

```
"v1.20.0-dev-185-ge8d0d2d923"
""
true
""
record SimulationResult
    resultFile = "/Users/u0110910/workspace/Modelica.Blocks.Examples.Filter_res.mat",
    simulationOptions = "startTime = 0.0, stopTime = 0.9, numberOfIntervals = 500, tolerance = 1e-06, method = 'dassl', fileNamePrefix = 'Modelica.Blocks.Examples.Filter', options = '', outputFormat = 'mat', variableFilter = '.*', cflags = '', simflags = ''",
    messages = "LOG_SUCCESS       | info    | The initialization finished successfully without homotopy method.
LOG_SUCCESS       | info    | The simulation finished successfully.
",
    timeFrontend = 0.085449207,
    timeBackend = 0.205186673,
    timeSimCode = 0.017255219,
    timeTemplates = 0.03251386100000001,
    timeCompile = 0.495335432,
    timeSimulation = 0.188640385,
    timeTotal = 1.025602167
end SimulationResult;
""
record SimulationResult
    resultFile = "",
    simulationOptions = "startTime = 0.0, stopTime = 4.0, numberOfIntervals = 500, tolerance = 1e-06, method = 'dassl', fileNamePrefix = 'Modelica.Blocks.Examples.PID_Controller', options = '', outputFormat = 'mat', variableFilter = '.*', cflags = '', simflags = ''",
    messages = "Failed to build model: Modelica.Blocks.Examples.PID_Controller",
    timeFrontend = 0.093452243,
    timeBackend = 0.07633056100000001,
    timeSimCode = 0.017951358,
    timeTemplates = 0.027575903,
    timeCompile = 0.425029329,
    timeSimulation = 0.0,
    timeTotal = 0.640480803
end SimulationResult;
"Error: Error building simulator. Build log: /usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller.o Modelica.Blocks.Examples.PID_Controller.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_functions.o Modelica.Blocks.Examples.PID_Controller_functions.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_records.o Modelica.Blocks.Examples.PID_Controller_records.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_01exo.o Modelica.Blocks.Examples.PID_Controller_01exo.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_02nls.o Modelica.Blocks.Examples.PID_Controller_02nls.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_03lsy.o Modelica.Blocks.Examples.PID_Controller_03lsy.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_04set.o Modelica.Blocks.Examples.PID_Controller_04set.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_05evt.o Modelica.Blocks.Examples.PID_Controller_05evt.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_06inz.o Modelica.Blocks.Examples.PID_Controller_06inz.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_07dly.o Modelica.Blocks.Examples.PID_Controller_07dly.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_08bnd.o Modelica.Blocks.Examples.PID_Controller_08bnd.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_09alg.o Modelica.Blocks.Examples.PID_Controller_09alg.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_10asr.o Modelica.Blocks.Examples.PID_Controller_10asr.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_11mix.o Modelica.Blocks.Examples.PID_Controller_11mix.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_12jac.o Modelica.Blocks.Examples.PID_Controller_12jac.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_13opt.o Modelica.Blocks.Examples.PID_Controller_13opt.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_14lnz.o Modelica.Blocks.Examples.PID_Controller_14lnz.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_15syn.o Modelica.Blocks.Examples.PID_Controller_15syn.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_16dae.o Modelica.Blocks.Examples.PID_Controller_16dae.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_17inl.o Modelica.Blocks.Examples.PID_Controller_17inl.c
/usr/bin/clang  -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0  -c -o Modelica.Blocks.Examples.PID_Controller_18spd.o Modelica.Blocks.Examples.PID_Controller_18spd.c
/usr/bin/clang -I. -o Modelica.Blocks.Examples.PID_Controller Modelica.Blocks.Examples.PID_Controller.o Modelica.Blocks.Examples.PID_Controller_functions.o Modelica.Blocks.Examples.PID_Controller_records.o Modelica.Blocks.Examples.PID_Controller_01exo.o Modelica.Blocks.Examples.PID_Controller_02nls.o Modelica.Blocks.Examples.PID_Controller_03lsy.o Modelica.Blocks.Examples.PID_Controller_04set.o Modelica.Blocks.Examples.PID_Controller_05evt.o Modelica.Blocks.Examples.PID_Controller_06inz.o Modelica.Blocks.Examples.PID_Controller_07dly.o Modelica.Blocks.Examples.PID_Controller_08bnd.o Modelica.Blocks.Examples.PID_Controller_09alg.o Modelica.Blocks.Examples.PID_Controller_10asr.o Modelica.Blocks.Examples.PID_Controller_11mix.o Modelica.Blocks.Examples.PID_Controller_12jac.o Modelica.Blocks.Examples.PID_Controller_13opt.o Modelica.Blocks.Examples.PID_Controller_14lnz.o Modelica.Blocks.Examples.PID_Controller_15syn.o Modelica.Blocks.Examples.PID_Controller_16dae.o Modelica.Blocks.Examples.PID_Controller_17inl.o Modelica.Blocks.Examples.PID_Controller_18spd.o -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0 -L\"/Users/u0110910/.openmodelica/libraries/Modelica 4.0.0+maint.om/Blocks\"    -Os -fPIC     -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc/c\" -I\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/include/omc\" -I. -DOPENMODELICA_XML_FROM_FILE_AT_RUNTIME -DOMC_MODEL_PREFIX=Modelica_Blocks_Examples_PID_Controller -DOMC_NUM_MIXED_SYSTEMS=0 -DOMC_NUM_LINEAR_SYSTEMS=0 -DOMC_NUM_NONLINEAR_SYSTEMS=1 -DOMC_NDELAY_EXPRESSIONS=0 -DOMC_NVAR_STRING=0 -L\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/lib//omc\" -L\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/lib\" -Wl,-rpath,\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/lib//omc\"  -Wl,-rpath,\"/Users/u0110910/workspace/OpenModelica/build_cmake/install_cmake/lib\"   -lSimulationRuntimeC -lOpenModelicaRuntimeC -lomcgc -llapack -lblas -lm
ld: can't write output file to 'Modelica.Blocks.Examples.PID_Controller' because that path is a directory
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [omc_main_target] Error 1
```

### Expected behavior
I was expecting both models to simulate. 

### Version and OS
 - OpenModelica Version `v1.20.0-dev-185-ge8d0d2d923`
 - Modelica `v4.0.0`
 - OS: MacOS with ARM

--- a/.gitignore
+++ b/.gitignore
@@ -21,4 +21,5 @@
 /OMSetup/
 /OpenModelicaSetup/
 /testsuite/libraries-for-testing/.ccache/
+/testsuite/flattening/modelica/ffi/FFITest/Resources/BuildProjects/gcc/libFFITestLib.dll
 trace.txt

--- a/OMCompiler/.gitignore
+++ b/OMCompiler/.gitignore
@@ -200,6 +200,8 @@ SimulationRuntime/c/RuntimeSources.mo
 SimulationRuntime/**/test_files/*.jar
 SimulationRuntime/OMSICpp/build_msvc/
 SimulationRuntime/cpp/build_msvc/
+SimulationRuntime/c/libOpenModelicaRuntimeC.dll
+SimulationRuntime/c/libSimulationRuntimeC.dll
 # Visual Studio 2015/2017 cache/options directory
 
 SimulationRuntime/OMSICpp/.vs/

--- a/OMEdit/OMEditLIB/Plotting/VariablesWidget.cpp
+++ b/OMEdit/OMEditLIB/Plotting/VariablesWidget.cpp
@@ -776,8 +776,16 @@ bool VariablesTreeModel::insertVariablesItems(QString fileName, QString filePath
   }
   // create hash based VariableNode
   VariableNode *pTopVariableNode = new VariableNode(variabledata);
+
   // sort the variables using natural sort
+#if defined(__APPLE__) // avoid crash on MacOS, see #9180
+  QMap<QString, QString> m;
+  for (auto variableName : variablesList)
+    m[StringHandler::cleanResultVariable(variableName)] = variableName;
+  variablesList = QStringList(m.values());
+#endif
   std::sort(variablesList.begin(), variablesList.end(), StringHandler::naturalSortForResultVariables);
+
   // remove time from variables list
   variablesList.removeOne("time");
   /* Fixes issue #7551

